src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -311,31 +311,30 @@ module Network_manager = struct
   let run_cmd_exn t prog args = run_cmd_exn t.testnet_dir prog args
 
   let create ~logger (network_config : Network_config.t) =
+    let%bind all_namespaces =
+      Cmd_util.run_cmd_exn "/" "kubectl"
```

Comment:
`kubectl get namespaces "-ojsonpath={.items[*].metadata.name}"` will give you a space separated list of names. Then you can `String.split` it and check to see if that list contains the namespace directly, avoiding the regex.